homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -79,6 +81,9 @@ class SensorType:
     "pressure": SensorType(
         unit=PRESSURE_HPA, icon=None, device_class=DEVICE_CLASS_PRESSURE
     ),
+    "load_power": SensorType(
```

Comment:
This seems to have snuck in. :wink: 

Please keep every PR focused on one thing.

Reply:
I removed it, will add it in a seperate PR that will also implement the complete switches after this one is done.